lua/diagnosticls-nvim/init.lua

```diff
@@ -2,59 +2,97 @@ local lspconfig = require 'lspconfig'
 local M = {}
 
 local diagnosticls_nvim_lsp_opts = {
-  root_dir = require 'lspconfig'.util.root_pattern('.git')
```

Comment:
I wanna keep the line indents to 2 spaces for now, I should add an editorconfig to normalize the indents